packages/notifi-frontend-client/lib/configuration/NotifiFrontendConfiguration.ts

```diff
@@ -196,10 +196,23 @@ export const envUrl = (env: NotifiEnvironment): string => {
     case 'Development':
       return 'https://api.dev.notifi.network/gql';
     case 'Local':
-      return 'https://localhost:5001/gql';
+      return 'http://localhost:5001/gql';
     case 'Production':
       return 'https://api.notifi.network/gql';
     case 'Staging':
       return 'https://api.stg.notifi.network/gql';
   }
 };
+
+export const envWSGraphqlUrl = (env: NotifiEnvironment): string => {
+  switch (env) {
+    case 'Development':
+      return 'ws://api.dev.notifi.network/gql';
+    case 'Local':
+      return 'ws://localhost:5001/gql';
+    case 'Production':
+      return 'ws://api.notifi.network/gql';
+    case 'Staging':
+      return 'ws://api.stg.notifi.network/gql';
+  }
+};
```

Comment:
I think it will be more efficient to combine this section with `envUrl`. Something like below:

```typescript
export const envUrl = (env: NotifiEnvironment, endpointType?: 'websocket' | 'http'): string => {
  switch (env) {
    case 'Development':
      return endpointType === 'websocket' ? 'ws://api.dev.notifi.network/gql' :  'https://api.dev.notifi.network/gql' : '';
    //  others ....
  }
};
```